lib/resources/dstu2/document_reference_create.yaml

```diff
@@ -0,0 +1,130 @@
+---
+field_name_base_url: http://hl7.org/fhir/DSTU2/documentreference-definitions.html#DocumentReference
+fields:
+- name: resourceType
```

Comment:
The link to the resourceType (and other fields defined as part of the core resource or buried in the referenced type) don't work since the links all assume they're on the DocumentReference page.